JLCoordinator/Sources/Presenter/PresenterImplementations/InitialPresenter.swift

```diff
@@ -24,5 +24,15 @@ public class InitialPresenter: InitialPresenting {
         window.rootViewController = viewController
         window.makeKeyAndVisible()
         notifyObserverAboutPresentation(of: viewController)
+
+        if animated {
+            UIView.transition(
+                with: window,
+                duration: 0.3,
+                options: .transitionCrossDissolve,
```

Comment:
Could we add a configuration possibility to modify the animation options and duration?

Reply:
Yes we can do that with a default fallback